### DISCUSSION TITLE
Rapid clicking very unresponsive

### DISCUSSION THREAD

**Discussion by tryashtar** [Mojang/minecraft-editor#38] [Bugs]
Here is me spam-clicking over and over while moving the mouse. As you can see, it's extremely unresponsive.
https://user-images.githubusercontent.com/26075577/223867448-9f3f7cbc-d8a4-4e87-81f5-0aeeadabfb2a.mp4

**Comment by dacowan** [comment]
Your FPS is super low there -- what spec machine are you running on, what resolution, how much RAM, etc?

**Reply by tryashtar** [reply]
My specs:
```
OS: Windows 11 Pro 10.0.22623 Build
Processor: AMD Ryzen 5 3600 6-Core Processor @ 3.60 GHz
RAM: 12.0 GB
Storage: Samsung SSD 970 EVO 500GB NVMe M.2 (93 GB free)
Graphics: NVIDIA GeForce RTX 2060, 1080p
```

**Reply by minecraftchloe** [reply]
Any other background processes that are causing FPS to tank?
